readme.md

```diff
@@ -56,10 +56,16 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
 - [7-Zip](http://www.7-zip.org/) (for deployment only)
 - [NSIS](http://nsis.sourceforge.net/) (for deployment only)
 
-### Mac:
+### macOS:
+- Xcode 8
+
+The program can also be built as a command line program using CMake. This type of build requires:
```

Comment:
It's worth flagging this so I don't forget, but the same "command line" phrasing was used here (so if we change it, we should change it here too)